src/diffusers/pipelines/stable_diffusion/convert_from_ckpt.py

```diff
@@ -1190,16 +1195,40 @@ def download_from_original_stable_diffusion_ckpt(
                     requires_safety_checker=False,
                 )
             else:
-                pipe = StableDiffusionPipeline(
-                    vae=vae,
-                    text_encoder=text_model,
-                    tokenizer=tokenizer,
-                    unet=unet,
-                    scheduler=scheduler,
-                    safety_checker=None,
-                    feature_extractor=None,
-                    requires_safety_checker=False,
-                )
+                if "LatentInpaintDiffusion" in original_config.model.target:
```

Comment:
```suggestion
                if hasattr(original_config, "model") and hasattr(original_config.model, "target") and "LatentInpaintDiffusion" in original_config.model.target:
```

Can we make it a bit more "defensive" here in case a weird config is passed ? :-) (maybe below as well)